### ISSUE TITLE
Regional extracts shapefile

### ISSUE THREAD

**Issue by lhorus** [mapsme/omim#10744]
I've been digging up Maps.Me's OSM extraction process, and one thing remains unclear: how are the regional extract's shapefiles defined? For instance, Peru is split into three sections - North, Lima, South - which don't correspond to any administrative level, my guess is this is done to avoid having massive files. However, where and how are these defined? Manually in a hidden file? Automatically using Splitter or similar?

**Comment by tatiana-yan** [comment]
You are right, it was done to avoid massive files. And it's not hidden, see data/borders.
